### DISCUSSION TITLE
Haier Appliances

### DISCUSSION THREAD

**Discussion by bokkoman** [ofalvai/home-assistant-candy#254] [Device support improvements]
I have a Haier washing machine and dryer.
They both don't work with the home assistant addon.
I put in the ip of a device, but tells me it can't find the key.
Any help appreciated.

**Comment by ofalvai** [comment]
Please see this section of the readme: https://github.com/ofalvai/home-assistant-candy#my-device-isnt-supported-can-you-help

**Reply by bokkoman** [reply]
> s section of

When trying the terminal commands from a PC, it doesn't work either.
